tests/middleware/test_middleware.py

```diff
@@ -0,0 +1,241 @@
+from unittest.mock import AsyncMock, call
+
+import pytest
+
+from kstreams import ConsumerRecord, StreamEngine, TestStreamClient, middleware
+
+
+@pytest.mark.asyncio
+async def test_middleware_stack_from_stream(
+    stream_engine: StreamEngine, dlq_middleware
+):
+    stream_name = "my-stream"
+
+    @stream_engine.stream(
+        "kstreams-topic",
+        name=stream_name,
+        middlewares=[middleware.MiddlewareFactory(dlq_middleware.middleware)],
```

Comment:
I understand in progress? since I don't see import of dlq_middleware?

Reply:
It comes from the fixtures (conftest.py)